sqlglot/expressions.py

```diff
@@ -1130,9 +1130,12 @@ class SetItem(Expression):
 class Show(Expression):
     arg_types = {
         "this": True,
+        "terse": False,
         "target": False,
         "offset": False,
+        "starts_with": False,
         "limit": False,
+        "from_": False,
```

Comment:
Let's just do `"from"` instead and then you can instantiate the expression using the `**{"from": ..., ....}` syntax